.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "🐛 Bug report"
+about: Submit a bug
+title: "[🐛]"
+labels: "🐛  bug"
+assignees: ''
+---
```

Comment:
I don't quite understand this table, how is it meant to be used?

Reply:
Apparently, it transforms into this table with requirements to the Issue look
https://github.com/snyk/snyk/blob/master/.github/ISSUE_TEMPLATE/bug-report.md